dynamo-neva-finetune.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+# nvidia-smi | grep 'python' | awk '{ print $5 }' | xargs -n1 kill -9
+
+TMPDIR=./foo-neva-train
+# profile="/tmp/neva-a100-dyn-thunder.nsys-rep"
+rm -fr ${TMPDIR}
+
+#source ~/env/bin/activate
+#
+#  --trace=cuda,nvtx,cublas,cudnn \
+#  --trace=cuda,nvtx,cublas,osrt,cudnn \
+#nsys profile \
+#  --force-overwrite=true \
+#  --output=${profile} \
+#  --opengl-gpu-workload=false \
+#  --stats=false \
+#  --show-output=true \
+#  --trace=cuda,nvtx,cublas,cudnn \
+#
+# nsys profile -w true -t cublas,cuda,nvtx,osrt -s cpu -c cudaProfilerApi -o dynamo_thunder_neva \
+
+# rm -f /tmp/graph*.log.txt
+
+
+# /scratch/nsight-systems-linux-public-DVS/target-linux-x64/nsys profile -w true -t cublas,cuda,nvtx,osrt -s cpu -c cudaProfilerApi -o dynamo_thunder_neva_finetune \
+
+HYDRA_FULL_ERROR=1 \
+THUNDER_ANNOTATE_TRACES=1 \
+NEMO_THUNDER_NEVA=dynamo \
+python3 \
+	./examples/multimodal/multimodal_llm/neva/neva_finetune.py \
+         --config-path=/scratch/thunder_neva/tfogal_nemo_neva \
```

Comment:
i'll see if we can use `./` for the path so that we don't have to force people to have hardcoded paths

Reply:
Ah yes. I'll check this as well if I can grab a node.